hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/Message.java

```diff
@@ -137,6 +140,13 @@ public String getId() {
         return Collections.unmodifiableList(additionalHeaders);
     }
 
+    public long updateAndGetCurrentMessageBackoff(SubscriptionPolicy subscriptionPolicy) {
+        currentMessageBackoff = currentMessageBackoff == -1 ? subscriptionPolicy.getMessageBackoff() :
```

Comment:
I'd recommend splitting it into several statements, since it's pretty tough on eyes at the moment :)